modules/ting_das/js/ting_das.js

```diff
@@ -0,0 +1,22 @@
+(function ($) {
+  "use strict";
+
+  // The click event on the button is blocked by Drupals ajax mechanism so we use ajaxSend to detect
+  // that a ajax call is happening and check if comes from pressing the order button
+  $(document).ajaxSend(function (e, xhr, settings) {
+    if ((typeof settings !== 'undefined') && (typeof settings.extraData !== 'undefined')) {
+      if (typeof settings.extraData._triggering_element_name !== 'undefined') {
+        var trigger = settings.extraData._triggering_element_name;
```

Comment:
If I remeber rigth it's posible to change the element name via the `#name = 'das-submit'` there by making the a little more sure that it's the right click event.